app/controllers/content.js

```diff
@@ -22,6 +22,12 @@ export default Ember.Controller.extend({
              this.set('authors', contributors));
     }),
 
+    applyMathJax: Ember.observer('model', function() {
```

Comment:
This is for the content page, so there is no ` node `, only the model. Since there is no editing on the content page, but it is dependent on a promise, this observer will fire once, when the model has been loaded.